brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
(side note: in general we try to stick to a convention of using get_<whatever:> for methods that perform database queries or other expensive calculation.)